File: cfgs/pythoncfgs/MINIMUM_OPERATIONS_MAKE_GCD_ARRAY_MULTIPLE_K.py
Nodes
1;statement;foo_1 = 0;8;
2;conditional;fun_1(foo_1);9;
3;statement;return foo_1;14;
4;conditional;foo_1[foo_2] != 1 and foo_3[foo_4] > foo_5;10;
5;statement;foo_1 = foo_2 + foo_3 - foo_4[foo_5];13;
6;statement;foo_1 = foo_2 + fun_1(foo_3[foo_4] % foo_5, foo_6 - foo_7[foo_8] % foo_9);11;
7;exit;;
Edges
0;2;
1;3;4;
2;7;
3;5;6;
4;2;
5;2;
6;

File: cfgs/java_cfgs_foo/MINIMUM_OPERATIONS_MAKE_GCD_ARRAY_MULTIPLE_K.java
Nodes
1;special;;;
2;statement;int result = 0;14;
3;statement;int i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;conditional;(foo_1[foo_2] != 1) && (foo_3[foo_4] > foo_5);18;
6;statement;result = foo_1 + fun_1(foo_2[foo_3] % foo_4, foo_5 - (foo_6[foo_7] % foo_8));19;
7;statement;result = (foo_1 + foo_2) - foo_3[foo_4];22;
8;statement;++i;17;
9;statement;return foo_1;25;
10;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;9;
4;6;7;
5;8;
6;8;
7;4;
8;10;
9;
